boa3_test/tests/compiler_tests/test_neo_types.py

```diff
@@ -315,146 +162,94 @@ def test_uint256_mismatched_type(self):
 
     # region ECPoint
 
-    def test_ecpoint_call_bytes(self):
-        path, _ = self.get_deploy_file_paths('ecpoint', 'ECPointCallBytes.py')
-        runner = BoaTestRunner(runner_id=self.method_name())
-
-        invokes = []
-        expected_results = []
+    def create_ecpoint(self, publickey: bytes) -> cryptography.ECPoint:
+        return cryptography.ECPoint(publickey, cryptography.ECCCurve.SECP256R1)
 
-        value = bytes(33)
-        invokes.append(runner.call_contract(path, 'ecpoint', value,
-                                            expected_result_type=bytes))
-        expected_results.append(value)
+    def ecpoint_to_array(self, ecpoint: cryptography.ECPoint) -> bytes:
+        return ecpoint.to_array().zfill(33).zfill(33).replace(b'0', b'\x00')
```

Comment:
Do you think this method could be used on other tests and it would be better to move it to BoaTestCase or another helper?

Reply:
There are no other tests that need ECPoint placeholders, that's why I put those methods in this class instead of BoaTestCase.
There are tests in test_crypto that also use ECPoint, but it's not a generic one